src/components/HelperErrorText/HelperErrorText.stories.mdx

```diff
@@ -89,17 +92,24 @@ import DSProvider from "../../theme/provider";
 ## Accessibility
 
 In the Reservoir Design System (DS), the `HelperErrorText` component is always
-used as a child component when composing more complex components. In the case
-of form components, the `HelperErrorText` component is associated with a
-related input field by using the `aria-describedby` attribute in the
-associated element. In this way, the content of the `HelperErrorText` component
-is made available to screenreaders.
-
-The example below is rendered using the `TextInput` component. The `TextInput`
-component uses the `HelperErrorText` component to render the red
-`"This is error text :("` below the input field and that text element is
-referenced with the `aria-describedby` attribute on the HTML input element.
-This type of association is handled automatically by all DS `Form Elements` components.
+used as a child component when composing more complex components.
+
+In the case of form components, the `HelperErrorText` component is associated
+with a related input field by using the `aria-describedby` attribute in the
+associated element. This pattern ensures that the content of the `HelperErrorText`
+component is made available to screenreaders.
+
+This component will always render at least an empty `<div>` element even if no
```

Comment:
The section for `ariaLive` mentions that the attribute is added to the persistent `<div>` element.  I'm wondering is that should be say something here as well.